subiquitycore/ui/views/network_configure_wlan_interface.py

```diff
@@ -151,7 +151,7 @@ def done(self, sender):
         if self.device.configured_ssid[0] is None and self.form.ssid.value:
             # Turn DHCP4 on by default when specifying an SSID for
             # the first time...
-            self.device.dhcp4 = True
+            self.device.config['dhcp4'] = True
```

Comment:
Hm, should we enable dhcp6 too here?

Most of IoT is ipv4 still, but it should at least try ipv6 in 2020, right?

Reply:
A brief data-point on this: Mythic's Pi cloud is entirely IPv6 only (with a proxy to provide web-access to IPv4-only clients) so I'd say it's certainly getting more important in the IoT space.